src/content/docs/mobile-monitoring/new-relic-mobile-ios/ios-sdk-api/current-session-id.mdx

```diff
@@ -26,7 +26,7 @@ Compatible with all agent versions.
 
 ## Description
 
-Returns the current session ID. This method is useful for consolidating monitoring of app data (not just New Relic data) based on a single session definition and identifier. For example, you might want to use the same identifier for marketing analytics or user analytics.
+Returns the current session ID. This method is useful for consolidating monitoring of app data (not just New Relic data) based on a single session definition and identifier. For example, you might want to use the same identifier for marketing analytics or user analytics. This session ID will change every time the app comes to the foreground. 
```

Comment:
@smalsam-newr  The phrase "...the app comes to the foreground" is a bit unclear. Can you explain what "coming to the foreground" means?

Reply:
@smalsam-newr Following up w/ you on this comment. Thanks!